carma_wm/src/WMListenerWorker.cpp

```diff
@@ -345,6 +350,7 @@ void WMListenerWorker::routeCallback(const cav_msgs::RouteConstPtr& route_msg)
 
     // After setting map evaluate the current update queue to apply any updates that arrived before the map
     bool more_updates_to_apply = true;
+    rerouting_flag_ = false;
```

Comment:
I'm concerned that adding this here may prevent the functionality where we only re-compute the routing graph when needed from working properly. Please double check that still behaves as expected.

Reply:
When `rerouting_flag`_ is false, all other nodes other than `route` will have `false` in `world_model_->setMap(world_model_->getMutableMap(), current_map_version_, rerouting_flag_);` . So it will not recompute as expected.
But those nodes will set their route from the the remainder of logic in `routeCallback`